test/unit/argument_parser/format_parse_validators_test.cpp

```diff
@@ -75,7 +75,7 @@ TEST(validator_test, fullfill_concept)
     EXPECT_TRUE(seqan3::validator<seqan3::detail::default_validator<int> &>);
 
     EXPECT_TRUE(seqan3::validator<seqan3::detail::default_validator<std::vector<int>>>);
-    EXPECT_TRUE(seqan3::validator<seqan3::arithmetic_range_validator>);
+    EXPECT_TRUE(seqan3::validator<seqan3::arithmetic_range_validator<int>>);
```

Comment:
Shouldn't this also work for `double`?
```suggestion
    EXPECT_TRUE(seqan3::validator<seqan3::arithmetic_range_validator<int>>);
    EXPECT_TRUE(seqan3::validator<seqan3::arithmetic_range_validator<double>>);
```